### PR TITLE
GH-14854: Make changes to .md pages

### LINKED ISSUE #14854: [Docs] Make changes to arrow/ and arrow/r/README.md
### Describe the enhancement requested

Make changes to `arrow/` and `arrow/r/README.md` to update the change in the issue tracking workflow.

### Component(s)

Documentation

### PR DESCRIPTION
**Pull request by AlenkaF**
This PR is part of the work to update the documentation as we are moving from Jira issue tracker to GitHub issues:
https://github.com/apache/arrow/issues/14822

cc @thisisnic 

closes https://github.com/apache/arrow/issues/14854

### CODE REVIEW

File: r/README.md [hunk 1]
```diff
@@ -93,13 +93,12 @@ In addition to the official [Arrow R package documentation](https://arrow.apache
...
-tracker](https://issues.apache.org/jira/projects/ARROW/issues). Create
-an account or log in, then click "Create" to file an issue. Select the
-project "Apache Arrow (ARROW)", select the component "R", and begin
-the issue summary with "[R]" followed by a space. For more
-information, see the "Report bugs and propose features" section of the
-[Contributing to Apache
+example on [GitHub issues](https://github.com/apache/arrow/issues/new/choose).
+Log in to you account and select the type of issue you want to create.
+Add a meaningful title prefixed with **`[R]`** followed by a space, the
+issue summary and select component **R** from the dropdown list.
```
Comment (thisisnic) [inline r/README.md:99]:
```suggestion
Log in to your account and select the type of issue you want to create.
Add a meaningful title prefixed with **`[R]`** followed by a space, the
issue summary and select component **R** from the dropdown list.
```

**Review by thisisnic — APPROVED** [review]
Thanks for updating this!

**Comment by AlenkaF** [comment]
R CI failures are unrelated.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -21,42 +21,35 @@
...
-among existing Arrow issues.
-
-When you create a new JIRA entry, please don't forget to fill the "Component"
-field.  Arrow has many subcomponents and this helps triaging and filtering
-tremendously.  Also, we conventionally prefix the issue title with the component
-name in brackets, such as "[C++] Crash in Array::Frobnicate()", so as to make
-lists more easy to navigate, and we'd be grateful if you did the same.
+The Arrow project uses GitHub as a bug tracker.  To report a bug, sign in to
+your GitHub account and select **Bug Report** in the
+[GitHub issues](https://github.com/apache/arrow/issues/new/choose).
```
Comment (jorisvandenbossche) [inline CONTRIBUTING.md:26]: "in the github issues" sounds incomplete. Maybe the "github issues form" or "github issues selector"?

Alternatively, we could also point people to https://github.com/apache/arrow/issues and direct them to click on "New issue" ?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -21,42 +21,35 @@
...
+
+To be assigned to an issue, add a comment "take: #ISSUE-NUMBER" to that issue.
```
Comment (jorisvandenbossche) [inline CONTRIBUTING.md:28]:
```suggestion
To be assigned to an issue, add a comment "take" to that issue.
```

(I _think_ just "take" is enough)

File: README.md [hunk 4]
```diff
@@ -93,12 +93,13 @@ integrations in other projects, we'd be happy to have you involved:
 - Join the mailing list: send an email to
   [dev-subscribe@arrow.apache.org][1]. Share your ideas and use cases for the
   project.
-- [Follow our activity on JIRA][3]
+- Follow our activity on [GitHub issues][3] and [GitHub labels][6]
```
Comment (jorisvandenbossche) [inline README.md:96]: Is the link to the labels needed? (you also can't follow or watch issues based on the label)

**Review by jorisvandenbossche — COMMENTED** [review]
Thanks! A few minor comments

File: CONTRIBUTING.md (hunk 2 above)
Comment (AlenkaF) [reply CONTRIBUTING.md:26]: Agree, would choose the second, pointing to issues and then clicking on "New issue".

File: README.md (hunk 4 above)
Comment (AlenkaF) [reply README.md:96]: No, not needed at all. I just found it nice to see the activity across different components.

**Review by domoritz — APPROVED** [review]
JS looks good

**Review by jorisvandenbossche — APPROVED** [review]

**Pull request merged**